download-data.sh

```diff
@@ -31,3 +39,6 @@ for file in "${FILES[@]}"
 do
   ln -sfn $RELEASE/$file data/$file
 done
+
+# Unzip any zip files in the data directory using the update flag
+unzip -u -d data data/*.zip 
```

Comment:
sure no problem, only add upzip to the end of the scrip. @jaclyn-taroni please review.